var/spack/repos/builtin/packages/plink2/package.py

```diff
@@ -34,6 +37,9 @@ class Plink2(MakefilePackage):
 
     build_directory = "2.0/build_dynamic"
 
+    def url_for_version(self, version):
+        return f"https://github.com/chrchang/plink-ng/archive/refs/tags/{version!s}.tar.gz"
```

Comment:
What happens for you if you prepend `v` as in:

```suggestion
        return f"https://github.com/chrchang/plink-ng/archive/refs/tags/v{version!s}.tar.gz"
```

Reply:
I noticed `spack stage plink2@2.0.0-a.6.9` attempts, in order: spack binary cache, spack mirror cache, the URL from this PR, then the URL with `v` prepended before it finally succeeds. It then succeeds from local cache on subsequent calls UNLESS I delete it.

 Making the above change attempts one less fetch on the new version.

Reply:
Will do. Thank you so much for the suggestion!